kmem.c

```diff
@@ -58,16 +59,19 @@ static void vmreadexit(void)
 
 static unsigned long get_kernel_symbol_address(const char *name)
 {
-	const unsigned long text_section_start = 0xffffffff98e00000;
-	const unsigned long text_section_end = 0xffffffff99c00e81;
+	const unsigned long in_data_section = (unsigned long)&jiffies; // Exported variable in the data section (D) used for counting ticks
 
+	bool found_symbol = false;
 	unsigned long offset;
-	for (offset = text_section_start; offset <= text_section_end; offset += 0x1)
+	for (offset = in_data_section; offset >= 0; offset -= 0x1)
```

Comment:
Pointers would always be aligned to the native word length, so it is not needed to step by just one single value. Stepping by `sizeof(long)` would speed things up by 4-8 times